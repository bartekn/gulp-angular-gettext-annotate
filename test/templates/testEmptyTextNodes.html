<div>
    <b>Only b tag should be annotated.</b>
</div>