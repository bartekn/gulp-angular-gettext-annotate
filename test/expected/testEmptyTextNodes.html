<div>
    <b translate>Only b tag should be annotated.</b>
</div>